server/fixtures/userValidationData.js

```diff
@@ -0,0 +1,37 @@
+/** Test cases for user validation */
```

Comment:
Please refactor this according to the TTL suggestion this morning. You can meet with @mosinmiloluwa01  for how to go about it if need be